README.md

```diff
@@ -102,7 +102,25 @@ search string, and don't try too hard to find a perfect pubkey. You only need
 something distinctive enough distinguish between a handful of VPN targets in
 a status display.
 
+## Dockerfile
+
+This repo contains a Dockerfile for running the binary so you
+don't have to install rust or other build tools.
+To use it:
+
+```
+docker build -t wgvanity .          # to build the container
+
+docker run --rm wgvanity "string"   # string for the "vanity address"
```

Comment:
If you run the container with `-it` flags you'll be able to stop it with Ctrl^C as usual, no need to open a new terminal.
Might be a bit easier approach.

Reply:
Actually, the `-it` doesn't do it. (I thought of that.) Ctl-C's seem to get swallowed...

Reply:
Sorry for assuming you didn't try that and not testing locally, my bad! I was able to reproduce now.

It seems the issue is related to [this SO post](https://stackoverflow.com/a/60812082). Running the container like that:
```
docker run --rm --init -it wgvanity Rich
```
makes it work for me. But it'd be probably better to "add explicit signal handling" to the app for SIGTERM.

Reply:
Bingo! That does the trick. I just updated the PR. THANKS!